lib/active_merchant/billing/gateways/sum_up.rb

```diff
@@ -139,11 +137,15 @@ def commit(action, post, method = :post)
       end
 
       def api_request(action, post, method)
-        begin
-          raw_response = ssl_request(method, live_url + action, post.to_json, auth_headers)
-        rescue ResponseError => e
-          raw_response = e.response.body
-        end
+        raw_response =
```

Comment:
💬 Style/comment:

A couple of things about this method:

1) `build_response` sounds like an action-method seems me that doesn't represent the actual content of the variable.
2) Is not super bad, but I'm not sure if is a good idea to be creative and invent a response that was not delivered by the gateway itself, maybe at some point the customer is going to go to support with a transcript and this text/message was never part of the gateway's response.

3) `format_multiple_errors` returns a structured error, if the gateway returns just 1 error, that is structure fits the one returned by `format_multiple_errors` something like this => 

```ruby
{
  error_code: "xxxx",
  message: 'xxxxxxxx,
  errors: errors
}
```